### ISSUE TITLE
Update chokidar - macOS fixes

### ISSUE THREAD

**Issue by goyney** [foreversd/forever-monitor#195]
Please consider updating chokidar to the latest version (this library is a full major version behind). They have just fixed several issues around macOS and Docker that would improve file watching.

**Comment by kibertoad** [comment]
@goyney AFAIR, main reason behind that was chokidar dropping compatibility with Node 6 in the major release. Considering that `forever` is run on legacy systems first and foremost, we have to be very conservative about dropping support for even old Node.js versions.

**Comment by kibertoad** [comment]
Double-checked - yup, 3.0.0 is Node 8+-only

**Comment by goyney** [comment]
I mean, that is what semver is for. Those instances wouldn't update? Seems like this could be a major version bump.

**Comment by kibertoad** [comment]
@goyney That is true. However, that would mean that those older instances will stop receiving any fixes at all, unless someone backports them, and doing any changes in forever is painful already.
Many systems that are running `forever` are using deprecated Node versions. Even when we dropped Node 4 there were complaints from users. 

Check these statistics: 
https://nodejs.org/metrics/summaries/version.csv
https://nodejs.org/metrics/summaries/version.png

Node 6 is still very much a thing (even though personally I would very much preferred if it wasn't)
In general `forever` is mostly a legacy software, meant for legacy systems, currently on life-support. For newer systems using pm2 or nodemon is a much better option.

**Comment by Hypnosphi** [comment]
FYI here's an NPM warning on clean install:

```
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
```

Here's how babel plans to deal with it without dropping node 6 support: https://github.com/babel/babel/pull/11560

**Comment by kibertoad** [comment]
@Hypnosphi OK, this is pretty smart. Would you be open to submitting a PR implementing the babel solution?

**Comment by kibertoad** [comment]
They mention that it doesn't work for Node 6, though?..
